packages/backend/src/index.ts

```diff
@@ -9,8 +9,14 @@ export * from './tokens/jwt';
 export * from './tokens/verify';
 export { constants } from './constants';
 
+// TODO: COR-164 improve type-safety
```

Comment:
❓ Can you please elaborate on this?

Reply:
@SokratisVidros I am thinking of using the type `DomainOrProxyUrl` wherever possible

```typescript
export type DomainOrProxyUrl = {
    proxyUrl?: never;
    domain: string;
} | {
    proxyUrl: string;
    domain?: never;
};
```